### PR TITLE
Frontend Section Bugfixes #387

### PR DESCRIPTION
**Pull request by mjashanks**
#387

### CODE REVIEW

File: packages/builder/src/builderStore/store/backend.js [hunk 1]
```diff
@@ -60,6 +60,10 @@ export const getBackendUiStore = () => {
       },
       select: model =>
         store.update(state => {
+          model =
```
Comment (shogunpurple) [inline packages/builder/src/builderStore/store/backend.js:63]: Should we just:

A) Always pass a modelId to this function
B) Always pass a model object to this function

**Review by shogunpurple — CHANGES_REQUESTED** [review]
We are reimplementing default browser behaviour through regex, parsing and mapping of CSS strings here. Have a look at this excerpt from the W3 spec:

Screenshot 2020-06-27 at 10 51 59

The browser is smart about what it actually does download. When the browser performs the layout step and builds its render tree, it knows which fonts are used on the page, and will only fetch the ones that are required. Have a look at this as well - https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization

In any case, I think we can just inject the `link` tags we need (the ones in the components, etc) into the head of the document, and we can allow people to add more fonts in future by pasting link tags into the builder, or allowing them to select one from google fonts which will then be added as further `link` tags to their apps `head`.

**Comment by mjashanks** [comment]
Thats really great to know, I had no idea :/

**Comment by mjashanks** [comment]
Reopening in another PR, as a --force pushed and this PR doesnt seem to pick that up

**Pull request closed without merging**